Index: strc-core/lib/Makefile.am
===================================================================
--- strc-core/lib/Makefile.am	(revision 17214)
+++ strc-core/lib/Makefile.am	(working copy)
@@ -52,3 +52,6 @@
 libstrc.rtree libstrc.c : $(strclib:.str=.rtree)
 	$(STRC)/bin/strc -c --library -i $(srcdir)/strc.str -o libstrc.rtree -O 2 --verbose 1 $(STRCFLAGS) $(STRINCLUDES)
 	rm libstrc.str
+
+libstrc.ctree:
+	$(STRC)/bin/strc -F --library -i $(srcdir)/strc.str -o libstrc.ctree -O 2 --verbose 1 $(STRCFLAGS) $(STRINCLUDES)
Index: stratego-libraries/lib/spec/Makefile.am
===================================================================
--- stratego-libraries/lib/spec/Makefile.am	(revision 17214)
+++ stratego-libraries/lib/spec/Makefile.am	(working copy)
@@ -98,3 +98,6 @@
   $(wildcard strategy/general*.str) \
   $(wildcard util/*.str) \
   $(wildcard util/config/*.str)
+
+libstratego-lib.ctree: $(fullstrategolib:.str=.rtree)
+	$(STRC_UNINSTALLED) $(STRCFLAGS) -F --library -i $(srcdir)/stratego-lib-posix-xsi.rtree -o libstratego-lib.ctree
Index: stratego-libraries/sglr/lib/Makefile.am
===================================================================
--- stratego-libraries/sglr/lib/Makefile.am	(revision 17214)
+++ stratego-libraries/sglr/lib/Makefile.am	(working copy)
@@ -55,3 +55,7 @@
 	@$(STRC_UNINSTALLED) -c --library -i stratego-sglr.rtree -o libstratego-sglr.rtree \
           --C-include "\"stratego_sglr.h\"" -O 2 --verbose 1 $(STRCFLAGS)
 	rm libstratego-sglr.str
+
+
+libstratego-sglr.ctree: $(sglrlib:.str=.rtree)
+	$(STRC_UNINSTALLED) $(STRCFLAGS) -F --library -i stratego-sglr.rtree -o libstratego-sglr.ctree -O 2 --verbose 1 $(STRCFLAGS)
